### PR TITLE
[Merged by Bors] - fix: use GH_TOKEN in maintainer merge

### PR DESCRIPTION
**Pull request by adomani**
This should really be the last fix!  :smile: 

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: .github/workflows/maintainer_merge_comment.yml [hunk 1]
```diff
@@ -27,8 +27,9 @@ jobs:
         id: determine_topic
         run: |
           ./scripts/get_tlabel.sh "${PR}" >> "$GITHUB_OUTPUT"
+        with:
+          GH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```
Comment (bryangingechen) [inline .github/workflows/maintainer_merge_comment.yml:31]: This needs to be back where it was below in the `env` section to work. (Same for the other instances.)

File: .github/workflows/maintainer_merge_comment.yml [hunk 2]
```diff
@@ -54,7 +55,7 @@ jobs:
       - name: Add label to PR
         uses: actions/github-script@v6
         with:
-          github-token: ${{secrets.GITHUB_TOKEN}}
+          GH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```
Comment (bryangingechen) [inline .github/workflows/maintainer_merge_comment.yml:58]: This one should be `github-token`

Comment (bryangingechen) [reply .github/workflows/maintainer_merge_comment.yml:57]:
```suggestion
          github-token: ${{secrets.GITHUB_TOKEN}}
```

File: .github/workflows/maintainer_merge_review.yml [hunk 3]
```diff
@@ -53,7 +53,7 @@ jobs:
       - name: Add label to PR
         uses: actions/github-script@v6
         with:
-          github-token: ${{secrets.GITHUB_TOKEN}}
+          GH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```
Comment (bryangingechen) [inline .github/workflows/maintainer_merge_review.yml:56]:
```suggestion
          github-token: ${{secrets.GITHUB_TOKEN}}
```

File: .github/workflows/maintainer_merge_review_comment.yml [hunk 4]
```diff
@@ -52,7 +52,7 @@ jobs:
       - name: Add label to PR
         uses: actions/github-script@v6
         with:
-          github-token: ${{secrets.GITHUB_TOKEN}}
+          GH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```
Comment (bryangingechen) [inline .github/workflows/maintainer_merge_review_comment.yml:55]:
```suggestion
          github-token: ${{secrets.GITHUB_TOKEN}}
```

**Review by bryangingechen — COMMENTED** [review]
These 3 steps are using `actions/github-script` which use an "input" named "github-script".

**Review by adomani — COMMENTED** [review]
Bryan's fix

**Review by adomani — COMMENTED** [review]
More fixes

**Pull request closed without merging**